dev/Breadcrumb/Breadcrumb.idl

```diff
@@ -0,0 +1,47 @@
+﻿namespace MU_XC_NAMESPACE
+{
+
+runtimeclass StackLayout;
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+[default_interface]
+runtimeclass BreadcrumbItemClickedEventArgs
+{
+    Object Item { get; };
+}
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+unsealed runtimeclass BreadcrumbItem : Windows.UI.Xaml.Controls.ContentControl
+{
+    BreadcrumbItem();
+}
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+unsealed runtimeclass BreadcrumbLayout : StackLayout
```

Comment:
You want to derive from NonVirtualizingLayout. StackLayout has other properties like orientation and spacing that we dont plan on supporting.